nvchecker_source/htmlparser.py

```diff
@@ -0,0 +1,31 @@
+# MIT licensed
+# Copyright (c) 2020 Ypsilik <tt2laurent.maud@gmail.com>, et al.
+# Copyright (c) 2013-2020 lilydjwg <lilydjwg@gmail.com>, et al.
+
+import re
+import sre_constants
+from lxml import html, etree
+
+from nvchecker.api import session, GetVersionError
+
+async def get_version(name, conf, *, cache, **kwargs):
+  key = tuple(sorted(conf.items()))
+  return await cache.get(key, get_version_impl)
+
+async def get_version_impl(info):
+  conf = dict(info)
+  encoding = conf.get('encoding', 'latin1')
+
+  res = await session.get(conf['url'])
+  body = html.fromstring(res.body.decode(encoding))
```

Comment:
lxml can parse the encoding declaration from the HTML, so we can delegate that to lxml if not specified.

```py
  encoding = conf.get('encoding')
  parser = html.HTMLParser(encoding=encoding)
  doc = html.fromstring(res.body, base_url=r.url, parser=parser)
```

It's possible to get the text encoding from `Content-Type` header, but maybe it's too much effort to parse that.

Reply:
You're right, I changed that.

Reply:
Did you forget to push your commits? I do not see any new commits yet.

Reply:
indeed, it's done.